tensorflow/python/keras/mixed_precision/experimental/autocast_variable_test.py

```diff
@@ -357,18 +371,18 @@ def test_assign_stays_in_true_dtype(self, distribution):
           dtypes.float16):
         # Variable should be increased, despite it appearing to be the same
         # float16 value.
-        self.assertEqual(1. + small_val,
-                         self.evaluate(x.assign(1. + small_tensor)))
```

Comment:
Now that `AutoCastDistributedVariable.assign` returns a `AutoCastDistributedVariable`, this assert doesn't seem to work anymore when inside a distribution strategy.